task/monorail-server/src/bsp/sidecar_bcd.rs

```diff
@@ -32,9 +33,20 @@ enum Trace {
     AnegCheckFailed(#[count(children)] VscError),
     Restarted10GAneg,
     Reinit,
+    UnlockUntil(u64),
+    LockingVLans,
+    AutomaticLock,
+    LockError(#[count(children)] VscError),
+    UnlockError(#[count(children)] VscError),
 }
 ringbuf!(Trace, 16, Trace::None);
 
+#[derive(Copy, Clone, Debug, PartialEq, Eq)]
```

Comment:
we shouldn't need to derive a `Debug` implementation for this, right? 
that's just extra strings in the binary we'll never use...
```suggestion
#[derive(Copy, Clone, PartialEq, Eq)]
```

Reply:
FWIW I'd be surprised if the debug impl were compiled into the image, but it's worth testing! Saving space is nice.

Reply:
Yeah, removing the derive doesn't change the code size at all here.